docs/rfcs/0001-ssl-transportation.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+
+Introduce the SSL/TLS protocol for secure data transport.
+
+# Motivation
+
+Encrypt the data transportation.
+
+# Usage explanation
+
+Define some global flags:
+
+```c++
+DEFINE_string(cert_path, "", "Path to cert pem.");
+DEFINE_string(key_path, "", "Path to cert key.");
+DEFINE_string(ca_path, "", "Path to trusted CA file.");
+DEFINE_bool(enable_ssl, false, "Whether enable ssl.");
+DEFINE_bool(enable_graph_ssl, false, "Whether enable ssl in graph server only.");
+DEFINE_bool(enable_meta_ssl, false, "Whether enable ssl in meta server only.");
```

Comment:
It's better to describe the flags by giving a table, but not only the original cpp code.

Reply:
Fixed.